spec/poros/tour_spec.rb

```diff
@@ -126,4 +126,126 @@
     expect(tour.name).to eq("Santa Fe Art District Tour")
     expect(tour.link).to be_a(String)
   end
+
+  it 'arranges coordinates by longitude' do
+    user = FactoryBot.create(:user)
+    user.street_arts.create!({
+      latitude: '39.758005',
+      longitude: '-104.985765',
+      address: '2532 Larimer Street',
+      city: 'Denver',
+      state: 'CO',
+      zipcode: '80205',
+      image_urls: "['/public/botanicare.png']",
+      description:'Her with plants'
+      })
+
+    user.street_arts.create!({
+      latitude: '39.765144',
+      longitude: '-104.979897',
+      address: '3196 Blake Street',
+      city: 'Denver',
+      state: 'CO',
+      zipcode: '80205',
+      image_urls: "['/public/pichi_and_avo.png']",
+      description: 'Hermes mural',
+      artist_name: 'PichiAvo',
+      instagram_handle: 'pichiavo'
+      })
+
+    user.street_arts.create!({
+      latitude: '39.763562',
+      longitude: '-104.978627',
+      address: '3128 Larimer Street',
+      city: 'Denver',
+      state: 'CO',
+      zipcode: '80205',
+      image_urls: "['/public/snake.png']",
+      description: 'Snake and moon',
+      artist_name: 'Lauren Napolitano',
+      instagram_handle: 'peptalks'
+      })
+
+    user.street_arts.create!({
+      latitude: '39.759987',
+      longitude: '-104.986479',
+      address: '2632 Blake Street',
+      city: 'Denver',
+      state: 'CO',
+      zipcode: '80205',
+      image_urls: "['/public/wolflady.png']",
+      description: 'Wolf lady mural'
+      })
+
+    tour = RinoTour.new
+
+    expected =
+    [["39.763562", "-104.978627"],
+    ["39.765144", "-104.979897"],
+    ["39.758005", "-104.985765"],
+    ["39.759987", "-104.986479"]]
```

Comment:
I just dropped some coordinates in google maps and rearranged them until it seemed like a logical sequence. Then I wrote the method and tested if it actually arranged the coordinates the way I wanted, and the methods to get the job done were surprisingly simple to write!

Reply:
nice work!